pkg/cluster/info.go

```diff
@@ -220,3 +222,36 @@ func MergeImageOverrides(imageOverrides map[string]string, localImageOverrides [
 
 	return imageOverrides, nil
 }
+
+func (c *Info) GetBroker(namespace string) (bool, error) {
+	broker := &v1alpha1.Broker{}
+	err := c.ClientProducer.ForGeneral().Get(
+		context.TODO(), controllerClient.ObjectKey{
+			Namespace: namespace,
+			Name:      brokercr.Name,
+		}, broker)
```

Comment:
To followup on the discussion re: moving this to `NewInfo`, it's problematic b/c we need the proper broker namespace which wouldn't be available in `NewInfo` for all commands up front.